homeassistant/components/template/config_flow.py

```diff
@@ -143,6 +155,28 @@ def generate_schema(domain: str, flow_type: str) -> vol.Schema:
                 )
             }
 
+    if domain == Platform.COVER:
+        schema |= _SCHEMA_STATE | {
+            vol.Inclusive(OPEN_ACTION, CONF_OPEN_AND_CLOSE): selector.ActionSelector(),
+            vol.Inclusive(CLOSE_ACTION, CONF_OPEN_AND_CLOSE): selector.ActionSelector(),
+            vol.Optional(STOP_ACTION): selector.ActionSelector(),
+            vol.Optional(CONF_POSITION): selector.TemplateSelector(),
+            vol.Optional(POSITION_ACTION): selector.ActionSelector(),
+            vol.Optional(CONF_TILT): selector.TemplateSelector(),
+            vol.Optional(TILT_ACTION): selector.ActionSelector(),
```

Comment:
Cover is quite tricky, because it can be controlled and report state according to position 0..100 or by open/close. Do we need both in the config flow, and if we do, can we explain better?
Is tilt a core feature which should be included in the config flow?